docs/deathmatch-setup-guide.txt

```diff
@@ -0,0 +1,137 @@
+
+To connect to the server, each other player will need to open the console and type "connect", followed by your local IP Address. For example, "connect 192.168.0.22". (Open the console either by pressing Esc on the main menu or by pressing ~ while in game.)
+
+Once everyone is in the server, each player can go to the "Online -> Setup" menu to set their player name, team name (if you have teamplay enabled), and player color.
+
+Once everyone is ready, you can enter the "restart" command in console to begin the match! Have fun!
+
+
+======================
+Useful Server Commands
+======================
+These commands must be typed into the console. Open the console either by pressing Esc on the main menu or by pressing ~ while in game.
+
+restart							Restarts the match
+map [MAP_NAME]					Changes the map. See the list of maps below.
+deathmatch [MODE]				Changes the deathmatch mode. See the list of modes below.
+teamplay [MODE]					Changes the teamplay mode. See teamplay modes below.
+timelimit [VALUE]				Changes the match time limit. Value is in minutes.
+fraglimit [VALUE]				Changes the number of frags needed to win.
+samelevel 1						Disable automatic map change after the match ends.
+serverinfo [SETTING] [VALUE]	Changes a server infokey setting to the chosen value. See infokeys below.
+
+
+========
+Map List
+========
+LibreQuake comes with 13 deathmatch maps. Change the map using the "map" command.
+
+map lqdm1	Solstice: A small castle map with a simple layout. This is a good map to start on.
+map lqdm2	Hyperborea: A white temple with jump pads. Great for large free-for-alls.
+map lqdm3	Torture Pit: An acid-flooded map with only lower-tier weapons.
+map lqdm4	Psychofuge: An ancient lava fortress. The dangerous lava-filled underbelly contains some of the best items!
+map lqdm5	Transport Tubes: An industrial facility with locking air tunnels that carry you around the map.
+map lqdm6	Ghost Quarter: A castle map where new weapons and armor types unlock every 40 seconds.
+map lqdm7	Boomstick Basement: A claustrophobic map with a labyrinthine structure.
+map lqdm8	Alichar Sector: A very tall map with a long elevator to reach the Red Armor.
+map lqdm9	Outlands: A temple map designed for 4v4 team deathmatch. Use the grenade launcher to unlock secret doorways!
+map lqdm10	Lucifer's Gambit: An evil temple with Lucifer's Protection as the main powerup.
+map lqdm11	The Wicked Night: A medium-sized map built around a central nexus.
+map lqdm12	Javelin: Contains a respawning key you can use to permanently unlock new rooms and powerups!
+map lqdm13	Death Provides No Return: A void map with jump pads and toggling windows.
+
+Additionally, all of the singleplayer levels are compatible with deathmatch. These maps are great if you have a player count of five or above since the gameplay spaces are much larger. If you want to switch to the third map in Episode 1, you would use the command "map e1m3".
```

Comment:
I believe the correct command would be `changelevel` to keep the server running. Can you confirm?

Reply:
It's different between different engines. In FTEQW both commands keep the server running.

Reply:
just change it to changelevel for engine constancy and then we should be good